server/src/admin/admin.gateway.ts

```diff
@@ -1,3 +1,4 @@
+import { UpdateEventDataDto } from './../../dist/client/update-event-data.dto.d';
```

Comment:
This is a really surprising import, you might want to update this to ./update-events.dto.ts.